src/test/runner.ts

```diff
@@ -109,14 +109,9 @@ export class Runner {
   async _flushOutput() {
     // Calling process.exit() might truncate large stdout/stderr output.
     // See https://github.com/nodejs/node/issues/6456.
-    //
-    // We can use writableNeedDrain to workaround this, but it is only available
-    // since node v15.2.0.
```

Comment:
Node.js 15+ is not relevant for us since we need to support Node.js 12+. The new fix also works in Node.js 12+.